<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Project</class>
 <widget class="QWidget" name="Project">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #F2F4F5;
}</string>
  </property>
  <widget class="QPushButton" name="btnAddProject">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>411</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #1ABC9C;
    color: white;
    font-size: 14pt;
    font-weight: bold;
    padding: 10px;
    border-radius: 10px;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    border: none;
}

QPushButton:hover {
    background-color: #16A085;
}
</string>
   </property>
   <property name="text">
    <string>ADD PROJECT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEditProject">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>411</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #1ABC9C;
    color: white;
    font-size: 14pt;
    font-weight: bold;
    padding: 10px;
    border-radius: 10px;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    border: none;
}

QPushButton:hover {
    background-color: #16A085;
}
</string>
   </property>
   <property name="text">
    <string>EDIT PROJECT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnProjectList">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>411</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #1ABC9C;
    color: white;
    font-size: 14pt;
    font-weight: bold;
    padding: 10px;
    border-radius: 10px;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    border: none;
}

QPushButton:hover {
    background-color: #16A085;
}
</string>
   </property>
   <property name="text">
    <string>PROJECTS LIST</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>181</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 18pt;
    font-weight: bold;
    color:rgba(4, 99, 80, 1);
    font-family: &quot;Segoe UI&quot;, sans-serif;
}
</string>
   </property>
   <property name="text">
    <string>PROJECT AREA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgba(4, 99, 80, 1);
    color:white;
    padding: 6px 14px;
    font-size: 12pt;
    font-weight: bold;
    border-radius: 6px;
    font-family: &quot;Segoe UI&quot;, sans-serif;
}

QPushButton:hover {
    background-color: #95a5a6;
}
</string>
   </property>
   <property name="text">
    <string> &lt;- Back to home</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
